### ISSUE TITLE
Wloss Loss Function

### ISSUE THREAD

**Issue by Dpw506** [dectrfov/Y-net#1]
Hi, Thank you very much for this work. I'm very interested in this direction, but from your code, I found a few errors in the Wloss function and I don't know how to fix them.
(i)  x0, x1, x2 = wt3(x)
      y0, y1, y2 = dw3(y)
Whether it should be:
                        x0, x1, x2 = dwt(x)
                        y0, y1, y2 = dwt(y)
Looking forward to your reply as soon as possible!Thank you.

**Comment by dectrfov** [comment]
Very thanks for your attention.
Yes, it is a typo.
I have fixed these bugs.

**Comment by Dpw506** [comment]
You are welcome. In addition, could you please provide your training code?

**Comment by dectrfov** [comment]
Sorry I cannot provide the training codes.
Perhaps you can refer my previous repo https://github.com/dectrfov/Wavelet-U-net-Dehazing 
and import model and loss function in train.py
